frontend/apps/filemanager/filemanager.lua

```diff
@@ -1287,6 +1287,7 @@ function FileManager:showSelectedFilesList()
     local menu = Menu:new{
         is_borderless = true,
         is_popout = false,
+        truncate_left = true,
```

Comment:
Maybe start instead of left? Pinging @poire-z about bidi stuff. (E.g., would start be deceptive if it's actually left, etc.)

Reply:
Everything is currently named "Left/Right" (ie. LeftContainer, align=left/right).
But "left" should everywhere (I hope) behave as "start" (and right as "end") by all the mirroring done by the widgets themselves when the UI is RTL.
So, we code in a LTR world, using left & right - and it should automagically work in the RTL world.

Reply:
> Everything is currently named "Left/Right"

Sure, my question is if we should keep adding to everything. :-)

Reply:
(We're not yet adding anything with this PR, just reusing :)
But I guess it's clearer having LeftContainer and RightContainer, than StartContainer and EndContainer :)